php/Admin/Admin.php

```diff
@@ -108,4 +110,50 @@ public function enqueue_scripts() {
 			$this->plugin->get_version()
 		);
 	}
+
+	/**
+	 * Modifies the block-renderer request as additional post params are not allowed
+	 *
+	 * @param mixed           $result Response to replace the requested version with. Can be anything a normal endpoint can return, or null to not hijack the request.
+	 * @param WP_REST_Server  $server Server instance.
+	 * @param WP_REST_Request $request Request used to generate the response.
+	 */
+	public function modify_render_request( $result, $server, $request ) {
+		if ( 'POST' !== $request->get_method()
+			|| ! str_starts_with( $request->get_route(), '/wp/v2/block-renderer/genesis-custom-blocks' ) ) {
+
+			return $result;
+		}
+
+		global $inner_block_request;
+		$json                = json_decode( $request->get_body(), true );
+		$inner_block_request = $json['attributes']['inner_blocks'];
+		$attributes          = [ 'attributes' => $json['attributes']['attributes'] ];
+		$request->set_body( wp_json_encode( $attributes ) );
+	}
+
+	/**
+	 * Includes the inner blocks in the render response from the block_renderer
+	 *
+	 * @param array $parsed_block The parsed block.
+	 */
+	public function modify_render_block_data_defaults( $parsed_block ) {
+		global $inner_block_request;
```

Comment:
To be honest, I'd rather we didn't pass a global between functions. Though there's probably no clean way to do this.